examples/zoom-control-directive/index.d.ts

```diff
@@ -1,18 +1,8 @@
 import 'reflect-metadata';
 import 'zone.js';
-import { ControlPosition } from '../../lib/index';
-import { AfterViewInit } from '@angular/core';
-export declare class AppComponent implements AfterViewInit {
-    positionStates: ControlPosition[];
-    position: ControlPosition;
-    zoomInText: string;
-    zoomInTitle: string;
-    zoomOutText: string;
-    zoomOutTitle: string;
-    private mapComponent;
-    constructor();
-    handlePositionEvent(event: Event): void;
-    ngAfterViewInit(): void;
+import { ExampleAppComponentBlueprint, IExampleProperties } from '../app-component-blueprint';
```

Comment:
we have to kick out the d.ts files from our repository. They are generated files and do not have to be placed into the repo... see #158